packages/suite/src/actions/wallet/accountSearchActions.ts

```diff
@@ -0,0 +1,22 @@
+import { Account } from '@wallet-types';
+import { ACCOUNT_SEARCH } from '@wallet-actions/constants';
+
+export type AccountSearchActions =
+    | {
+          type: typeof ACCOUNT_SEARCH.SET_COIN_FILTER;
+          payload: Account['symbol'] | undefined;
+      }
+    | {
+          type: typeof ACCOUNT_SEARCH.SET_SEARCH_STRING;
+          payload: string | undefined;
+      };
+
+export const setCoinFilter = (payload: Account['symbol'] | undefined) => ({
```

Comment:
`payload?: Account['symbol']`

Reply:
:D I don't know why I always subconsciously prefer this syntax. I will update it.

Reply:
oh, there is a slight difference. Syntax I used won't allow calling setCoinFilter without an arg (`setCoinFilter ()` throws An argument for 'payload' was not provided), requested change will allow it just fine. I like that the argument needs to be passed even if it is undefined (it nicely tells a story that we are really setting undefined), but I don't have a strong opinion on it. Should I proceed updating the code to use `?`?

Reply:
ah, ok so leave it as is then. it was only a nitpick

Reply:
too late! updated! 😁